akka-cluster/src/main/mima-filters/2.5.10.backwards.excludes

```diff
@@ -0,0 +1,5 @@
+# #24622 wire compat regression
+ProblemFilters.exclude[FinalClassProblem]("akka.cluster.protobuf.ClusterMessageSerializer")
+ProblemFilters.exclude[FinalMethodProblem]("akka.serialization.SerializerWithStringManifest.includeManifest")
```

Comment:
Are these needed? `SerializerWithStringManifest` doesn't seem to be changed and also seems like a real compatibility problem.

Reply:
Oh, that is weird indeed, good catch I think I zombie-copy-pasted-those. That should not have changed!

Reply:
But they do fail MiMa, those methods has been final since 2015, I have no idea what is going on here. 

Why is mima even checking `akka.serialization` types when i run `akka-cluster/mimaReportBinaryIssues`? Running mima against `akka-actor` does not report anything. O.o

Reply:
```
[info] akka-cluster: found 2 potential binary incompatibilities while checking against com.typesafe.akka:akka-cluster_2.11:2.5.0  (filtered 65)
[error]  * method includeManifest()Boolean in class akka.serialization.SerializerWithStringManifest is declared final in current version
[error]    filter with: ProblemFilters.exclude[FinalMethodProblem]("akka.serialization.SerializerWithStringManifest.includeManifest")
[error]  * method fromBinary(Array[Byte],scala.Option)java.lang.Object in class akka.serialization.SerializerWithStringManifest is declared final in current version
[error]    filter with: ProblemFilters.exclude[FinalMethodProblem]("akka.serialization.SerializerWithStringManifest.fromBinary")
```

I have no clue here, those types are not in that jar-file. What is MiMa smoking?